manifests/init.pp

```diff
@@ -22,9 +22,7 @@
     case $::operatingsystem {
       'debian',
       'ubuntu': {
-        package { 'apt-transport-https':
-          ensure => latest,
-        }
+        ensure_packages('apt-transport-https')
```

Comment:
Took keep the same functionality, should it not be "{ ensure  => 'installed' }"

This should also be converted to ensure_packages:
```
       'OEL': {
         package { 'pygpgme':
           ensure => latest,
         }
````